packages/components/src/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-console */
+import React, { Dispatch, SetStateAction, useEffect, useState } from "react"
+import classNames from "classnames"
+import { useDropzone, FileRejection, DropzoneOptions } from "react-dropzone"
+import { ACCEPTED_FILE_TYPES, DEFAULT_MAX_FILE_SIZE_40KB, DROPZONE_AUTOMATION_ID } from "./constants"
+import { DefaultMessage } from "./sub-components/DefaultMessage/DefaultMessage"
+import { DropzoneErrors } from "./sub-components/DropzoneError/DropzoneError"
+import { DropzoneIcon } from "./sub-components/DropzoneIcon/DropzoneIcon"
+import { OnDragMessage } from "./sub-components/OnDragMessage/OnDragMessage"
+import { SelectedFileMessage } from "./sub-components/SelectedFileMessage/SelectedFileMessage"
+
+type DropzoneProps = {
```

Comment:
What are you thoughts on extending or `Pick`ing some of the `InputHTMLAttributes` as well. This can allow attributes like `name` when used inside of a `form` element being passed down to the dropzone component as well